interface-definitions/vpn_openconnect.xml.in

```diff
@@ -69,6 +69,30 @@
                       <valueless/>
                     </properties>
                   </leafNode>
+                  <leafNode name="cert">
+                    <properties>
+                      <help>Use certificate based authentication</help>
+                      <valueHelp>
+                        <format>cn</format>
+                        <description>OID 2.5.4.3 - Common Name</description>
```

Comment:
My feeling is that these should be sub-nodes like `cert cn oc.example.com`. That way it would be more obvious to config readers what field of the cert it's supposed to match, and it will be possible to add separate constraints for CNs and UIDs.

I also don't know if help strings should mention OIDs. End users normally talk about symbolic names like CN, I can't see how OIDs could be important or useful here in help strings.